### ISSUE TITLE
How to contribute my local changes

### ISSUE THREAD

**Issue by marco-aa** [Fab1en/rocket-chat-rest-client#8]
Dear Devs,

i made a local clone and created a local branch and made my changes/contributions.
(Added update method for Userclass and added optional field customFields for update and create)

How can i contribute to you/here ? Normaly i push my branch and make a pull request.

**Comment by Fab1en** [comment]
Great ! Go for it, feel free to make pull requests.
There is no guideline for contributors, we would have to build one together.

**Comment by marco-aa** [comment]
I would like to :)
Do i have to fork first and push my changes into my fork and than i have to make a pull request ? Or is there a easyer way - because i'm not a member ob Fab1en-rocket-chat-rest-client git repository ?

**Comment by Fab1en** [comment]
Yes, you have to fork first, push your changes to your own fork and then make the pull request. That's what @alekseykuleshov did (he seams to have deleted his repo afterwards). I found [this question on Stack Overflow](https://stackoverflow.com/questions/14821583/pull-request-without-forking) that would perhaps help you.

**Comment by marco-aa** [comment]
Worked like charme - done ;)
